### PR TITLE
[State transitions] Certify record

### PR DESCRIPTION
**Pull request by Zangetsu101**

### CODE REVIEW

File: packages/commons/src/fhir/transformers/index.ts [hunk 1]
```diff
@@ -2698,12 +2699,15 @@ const builders: IFieldBuilders = {
             if (!hasAffidavit) {
               relatedPersonResource.extension.push({
                 url: `${OPENCRVS_SPECIFICATION_URL}extension/relatedperson-affidavittype`,
+                //@ts-ignore
```
Comment (Zangetsu101) [inline packages/commons/src/fhir/transformers/index.ts:2702]: I will be removing the certificate related fhir-builders in the "issue-record" PR

File: packages/workflow/src/records/fhir.ts [hunk 2]
```diff
@@ -807,7 +885,130 @@ export async function sendBundleToHearth(
...
+            reference: sectionEntry.reference
+          }
+        }
+        const savedReference = findSavedReference(
+          sectionEntry.reference,
+          resourceBundle,
+          responseBundle
+        )
+        if (!savedReference) {
+          throw internal(
```
Comment (rikukissa) [inline packages/workflow/src/records/fhir.ts:940]:
```suggestion
          throw new Error(
```

Maybe better to just throw generic errors from this abstraction level so http related concepts are kept on the handler-level.

File: packages/workflow/src/records/fhir.ts [hunk 3]
```diff
@@ -807,7 +885,130 @@ export async function sendBundleToHearth(
...
+  const indexInResponseBundle = resourceBundle.entry.findIndex(
+    (entry) => entry.fullUrl === temporaryReference
+  )
+  if (indexInResponseBundle === -1) {
+    return null
+  }
+  return responseBundle.entry[indexInResponseBundle].response.location
+}
+
+function toSavedComposition(
```
Comment (rikukissa) [inline packages/workflow/src/records/fhir.ts:916]: Would the functions in this file make sense in the `commons` package?

Comment (Zangetsu101) [reply packages/workflow/src/records/fhir.ts:916]: I was thinking the same but they should not be needed anywhere other than workflow which defeats the purpose

**Pull request merged**